File: StreamingProduct/src/main/scala/com/domain/CourseClickCount.scala
package com.domain

/**
  * Created by Administrator on 2018/11/24.
  */
/**
  *
  * @param day_course 对应的就是HBase中的rowkey
  * @param click_count 对应rowkey的访问总数
  */
case class CourseClickCount (day_course:String,click_count:Long)

File: WebSpring/src/main/java/com/imooc/domain/CourseClickCount.java
package com.imooc.domain;

import org.springframework.stereotype.Component;

/**
 * Created by Administrator on 2018/9/26.
 * 实战课程访问数量实体类
 */
@Component
public class CourseClickCount {
    private String name;
    private long value;

    public long getValue() {
        return value;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setValue(long value) {
        this.value = value;
    }
}
